src/main/resources/bidder-config/visx.yaml

```diff
@@ -0,0 +1,20 @@
+adapters:
+  visx:
+    enabled: false
+    endpoint: https://t.visx.net/s2s_bid?wrapperType=s2s_prebid_standard
+    pbs-enforces-gdpr: true
+    deprecated-names:
+    aliases:
+    meta-info:
+      maintainer-email: service@yoc.com
+      app-media-types:
+      site-media-types:
+        - banner
+      supported-vendors:
+      vendor-id: 0
+    usersync:
+      url: https://t.visx.net/s2s_sync?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&r=
```

Comment:
Wrong placeholders. Please note that Go placeholders are different from the ones we use in Java so you always need to replace (or modify) them. Should be `{{gdpr}}` and `{{gdpr_consent}}` correspondingly.